redis/src/test/scala/zio/redis/ConnectionSpec.scala

```diff
@@ -159,6 +159,13 @@ trait ConnectionSpec extends BaseSpec {
         },
         testM("PING with input") {
           ping(Some("Hello")).map(assert(_)(equalTo("Hello")))
+        },
+        testM("PING with a string argument will not lock executor") {
+          ping(Some("Hello with a newline\n")).map(assert(_)(equalTo("Hello with a newline\n")))
```

Comment:
Is everything going to be ok if in the end would be \r ?

Reply:
You are right, as mentioned early, if we split only crlf, then we can recompose the exact value of the bulk string, avoiding an ambiguity between Lf and CrLf